### ISSUE TITLE
An option to download as images, in a structured way (or at least keep the temp)

### ISSUE THREAD

**Issue by jackblk** [leotaku/kojirou#48]
First of all, awesome project. This produces the highest quality file from Mangadex to Kindle, which is awesome.

TL;DR: an option to save to images in a structured directory

My use case is that I want to use Send To Kindle (STK) feature. It doesn't allow `azw3` file, so I have to convert `azw3` file to `epub`. However, this converted `epub` file will add a huge white margin to top & bottom because of Send To Kindle conversion.

To circumvent this, I have to download the whole manga and structure it this way:

```
.
└── Manga Title
    └── cover.jpg
    └── Volume 1: Vol Title
        └── Chapter 1: Chapter Title
            └── 1.jpg
            └── 2.jpg
            └── 3.jpg
            └── ...
```

Then use https://github.com/ciromattia/kcc/ to convert it to epub, then use Send To Kindle (STK). KCC is the only tool that produces an `epub` file that has no white spaces after STK.

I'm using another tool to download, however I think kojirou is the best right now.

It would be great if we have an option to just save temp files before packing it to `azw3` file. I can write my own tool to manage the directory structure and convert.

**Comment by leotaku** [comment]
Hi! Thanks for the suggestion and contribution. However, I am not sure if this is something I would like to add to this project.

As far as I know, there are many good MangaDex downloaders that will output image files in the specified directory layout, which tools like KCC can then use to generate e-books. Why do you think something like this should also be added to Kojirou?

**Comment by jackblk** [comment]
> there are many good MangaDex downloaders that will output image files in the specified directory layout

The only one that I'm aware of is https://github.com/mansuf/mangadex-downloader, but it's a lot slower since it's not using async to download.

This can be an option for Kojirou, just to save raw/temp images. Also it can be reused for `--disk` option.
